go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/spdk/sma-goapi
+
+go 1.19
+
+require (
+	google.golang.org/grpc v1.55.0
+	google.golang.org/protobuf v1.30.0
+)
+
+require (
+	github.com/golang/protobuf v1.5.3 // indirect
```

Comment:
@askervin  Is't golang/protobuf is deprecate? Instead we should use  `google.golang.org/protobuf` I guess.

Reply:
Okay got it. These go dependencies are coming from spdk. Please ignore my comment.